recipes/influxdb-cpp-rest/all/conanfile.py

```diff
@@ -52,6 +53,38 @@ def source(self):
 
     def validate(self):
         check_min_cppstd(self, 20)
+        # std::format requires:
+        # - GCC 13+ (not available in GCC 11-12)
+        # - Clang 14+
+        # - MSVC 19.29+ (VS 2019 16.10+)
+        if self.settings.compiler == "gcc":
+            # Extract major version number for comparison
+            gcc_version = str(self.settings.compiler.version)
+            gcc_major = int(gcc_version.split('.')[0])
+            if gcc_major < 13:
+                raise ConanInvalidConfiguration(
+                    f"influxdb-cpp-rest requires GCC 13+ for std::format support. "
+                    f"Current version: {self.settings.compiler.version}"
+                )
+        elif self.settings.compiler == "clang":
+            # Extract major version number for comparison
+            clang_version = str(self.settings.compiler.version)
+            clang_major = int(clang_version.split('.')[0])
+            if clang_major < 14:
+                raise ConanInvalidConfiguration(
+                    f"influxdb-cpp-rest requires Clang 14+ for std::format support. "
+                    f"Current version: {self.settings.compiler.version}"
+                )
+        elif self.settings.compiler == "msvc":
+            # MSVC version is stored as "191", "192", etc. (19.1 = 191, 19.29 = 1929)
+            # We need at least 192 (19.29)
+            msvc_version = str(self.settings.compiler.version)
+            msvc_numeric = int(msvc_version)
+            if msvc_numeric < 192:
+                raise ConanInvalidConfiguration(
+                    f"influxdb-cpp-rest requires MSVC 19.29+ (VS 2019 16.10+) for std::format support. "
+                    f"Current version: {self.settings.compiler.version}"
+                )
```

Comment:
Should it not be Clang `15` and Apple Clang `14`?
Also, we could refactor these lines as follows:

```python
        minimum_compiler_versions = {"gcc": 13, "clang": 15, "apple-clang": 14, "msvc": 192}
        minimum_version = minimum_compiler_versions.get(str(self.settings.compiler))
        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
            raise ConanInvalidConfiguration(f"Requires {self.settings.compiler} >= {minimum_version} for std::format support")
```